src/erlcloud_kinesis.erl

```diff
@@ -560,3 +564,117 @@ split_shards(StreamName, ShardToSplit, NewStartingHashKey) ->
 split_shards(StreamName, ShardToSplit, NewStartingHashKey, Config) when is_record(Config, aws_config) ->
   Json = [{<<"StreamName">>, StreamName}, {<<"ShardToSplit">>, ShardToSplit}, {<<"NewStartingHashKey">>, NewStartingHashKey}],
   erlcloud_kinesis_impl:request(Config, "Kinesis_20131202.SplitShard", Json).
+
+%%------------------------------------------------------------------------------
+%% @doc
+%% Kinesis API: AddTagsToStream
```

Comment:
Can you use a link, like [http://docs.aws.amazon.com/kinesis/latest/APIReference/API_AddTagsToStream.html](http://docs.aws.amazon.com/kinesis/latest/APIReference/API_AddTagsToStream.html), on all your references? I don't understand the divergence from the module style here.